model/src/w3gridmd.F90

```diff
@@ -2460,33 +2460,25 @@ SUBROUTINE W3GRID()
        B_JGS_NLEVEL = JGS_NLEVEL
        B_JGS_SOURCE_NONLINEAR = JGS_SOURCE_NONLINEAR
 
-       IF ((EXPFSN .eqv. .FALSE.).and.(EXPFSPSI .eqv. .FALSE.)     &
-           .and.(EXPFSFCT .eqv. .FALSE.)                           &
-           .and.(IMPFSN .eqv. .FALSE.)                             &
-           .and.(EXPTOTAL .eqv. .FALSE.)                           &
-           .and.(IMPTOTAL .eqv. .FALSE.)) THEN
-         EXPFSN=.TRUE. ! This is the default scheme ...
-       END IF
        nbSel=0
 
-       IF (EXPFSN) nbSel=nbSel+1
-       IF (EXPFSPSI) nbSel=nbSel+1
-       IF (EXPFSFCT) nbSel=nbSel+1
-       IF (IMPFSN) nbSel=nbSel+1
-       IF (IMPTOTAL) nbSel=nbSel+1
-       IF (EXPTOTAL) nbSel=nbSel+1
+       IF (EXPFSN)   nbSel = nbSel+1
+       IF (EXPFSPSI) nbSel = nbSel+1
+       IF (EXPFSFCT) nbSel = nbSel+1
+       IF (IMPFSN)   nbSel = nbSel+1
+       IF (IMPTOTAL) nbSel = nbSel+1
+       IF (EXPTOTAL) nbSel = nbSel+1
 
        IF (GTYPE .EQ. UNGTYPE) THEN
          IF (nbSel .ne. 1) THEN
            WRITE(NDSE,*) ' *** WAVEWATCH III ERROR IN WW3_GRID:'
            IF (nbSel .gt. 1) THEN
-             WRITE (NDSE,*) 'More than one scheme selected'
+             WRITE (*,*) 'MORE THAN ONE UNSTRUCTURED SCHEME SELECTED'
```

Comment:
Should this and the next write statement be reverted to be written to NDSE?

Reply:
We can delete this one.